swarm/network/simulation/simulation.go

```diff
@@ -164,17 +164,6 @@ var maxParallelCleanups = 10
 func (s *Simulation) Close() {
 	close(s.done)
 
-	// Close all connections before calling the Network Shutdown.
-	// It is possible that p2p.Server.Stop will block if there are
-	// existing connections.
-	for _, c := range s.Net.Conns {
```

Comment:
So is that not an issue anymore, that `Stop` may block?

Reply:
It does not seem to be an issue anymore, and  the connection are stopped anyway in p2p.Server, so this code is redundant